src/main/java/io/cryostat/discovery/KubeApiDiscovery.java

```diff
@@ -295,28 +295,33 @@ private void pruneOwnerChain(DiscoveryNode nsNode, Target target) {
             return;
         }
 
-        DiscoveryNode child = target.discoveryNode;
-        while (true) {
-            DiscoveryNode parent = child.parent;
+        QuarkusTransaction.joiningExisting()
```

Comment:
Just wondering: Do we need to remove the `QuarkusTransaction.joinExisting` here? Or if not, what does it mean to have nested transactions like this?

https://github.com/cryostatio/cryostat3/blob/12b97bf7746a72a965dc168c23c4fae69ec19953/src/main/java/io/cryostat/discovery/KubeApiDiscovery.java#L219

Reply:
Make sense! Thanks for explaining :D